### ISSUE TITLE
IAM CreateUser tags are not stored

### ISSUE THREAD

**Issue by adam-fowler** [localstack/localstack#1673]
Tags attached to an IAM user are not saved. Create a user with attached tags. The response from the create user doesn't include the tags and when you run a GetUser the tags are not included.
```
aws-sdk-swift % aws --endpoint-url http://localhost:4593 iam create-user --user-name test-user --tags Key=test,Value=hello
```
produces output
```
{
    "User": {
        "Path": "/",
        "UserName": "test-user",
        "UserId": "peldynzrvm2tofi6nmmr",
        "Arn": "arn:aws:iam::000000000000:user/test-user",
        "CreateDate": "2019-10-23T08:46:31.305Z"
    }
}
```
The GetUser call produces exactly the same output
```
aws --endpoint-url http://localhost:4593 iam get-user --user-name test-user
```

I am running localstack as follows
```
docker run -d -p 4593:4593 -e SERVICES='iam' localstack/localstack
```

**Comment by whummer** [comment]
Thanks for reporting @adam-fowler . This should be fixed in #2256 - can you please give it a try with the `latest` version? Please report here if the problem persists. Thanks

**Comment by adam-fowler** [comment]
Hi @whummer there's a discrepancy between what AWS returns and what this PR returns. AWS serializes the Tags array as follows
```
<Tags><member><Value>value</Value><Key>tag</Key></member><member><Value>value2</Value><Key>tag2</Key></member></Tags>
```
 while this PR serializes it as 
```
<Tags><Tag><Key>tag</Key><Value>value</Value></Tag><Tag><Key>tag2</Key><Value>value2</Value></Tag></Tags>
```

See https://docs.aws.amazon.com/IAM/latest/APIReference/API_User.html for REST api details
